src/plots/font_attributes.js

```diff
@@ -47,6 +47,84 @@ module.exports = function(opts) {
             valType: 'color',
             editType: colorEditType
         },
+
+        weight: {
+            editType: editType,
+            valType: 'enumerated',
```

Comment:
Is there a reason this is implemented as an enum rather than following the CSS convention of accepting `normal`, `bold`, or a value 1-1000?

Reply:
Addressed by 6095b93 and 28087ee.